devel/apache-ant-1.9/Portfile

```diff
@@ -21,9 +21,11 @@ platforms               darwin freebsd
 distname                apache-ant-${version}-bin
 master_sites            apache:ant/
 master_sites.mirror_subdir        binaries
-checksums               md5    565ffd791def7f22936311e9aa3f8058 \
-                        sha1   570005afd7c080436742e128b07eb97d50178d53 \
-                        sha512 3c0cc564c27483eb526e004b31b211dd689aa24a2aa7097fc11bc5304771e49124a657203eeec748a70a6bfa6f2c050d5647b37a69e5eb02843fff7a0f7646ea
+# Remember also to update the checksums in the source variant below.
+checksums               sha1    570005afd7c080436742e128b07eb97d50178d53 \
```

Comment:
sha1 should probably go too, since collisions have been found.

Reply:
I deliberately left it in to demonstrate that the file has not changed. I would certainly recommend removing sha1 the next time the port's version is updated, since the other checksum types suffice. Leaving it in does pose any risk of file compromise, since more than one checksum type is used.

Reply:
Maybe add comments to this effect?

Reply:
I'm not a big fan of adding comments for generally-known information. All ports should use modern checksum types, but that doesn't mean we should add a comment to all ports that aren't doing so yet.

Reply:
Someone might take a senior member of the project committing something with an old checksum as documentation of a different policy.